senior-knights/course-schedulizer
# Scroll to Section

To Test:
- Try updating sections, adding section from an existing section, adding completely new sections, and adding non-teaching loads (where applicable try doing so on both the faculty schedule, room schedule, and faculty loads tabs to make sure it works everywhere)
- Upon performing any of these actions, it should scroll to the section/load which was added/updated (or the instructor's row if you are on the faculty loads tab)
- Notice that instructor names and rooms are now sorted alphabetically
- Probably should try changing a section from one term to another (this should switch terms and then scroll to the section)

Notes:
- If there are multiple rooms/instructors for a given section, the scrolling will need to make a choice about which to scroll to, and it may not end up choosing the one you clicked (tries to choose the last one)
- The faculty loads page will scroll that instructor's row to the center of the screen (or close as close to the center as possible), but the room/faculty schedule tabs will just scroll that section into view because if I try to scroll to the center, it causes graphical issues

Closes #150 
Also Closes #132
